tests/test_repeat_request.py

```diff
@@ -31,3 +31,13 @@ def test_create_repeat_request_end_and_count_defined(test_client):
         test_client.create_repeat_request(
             geometry=mock_geojson, name="test", repeat_end="PANDA", repetition_count=12345
         )
+
+
+def test_create_repeat_request_with_valid_contract_id(test_client, authed_repeat_request_mock):
+    repeat_request = test_client.create_repeat_request(geometry=mock_geojson, name="test", contract_id="contract-123")
+    assert repeat_request == post_mock_responses("/repeat-requests")
+
+
+def test_create_repeat_request_with_invalid_contract_id(test_client, authed_repeat_request_mock):
+    with pytest.raises(ContractNotFoundError):
```

Comment:
note: I created and merged https://github.com/capellaspace/console-client/pull/117 based on this

which will result in an explicit `ValidationError` for this test case (if it were to hit the real api"

`capella_console_client.exceptions.ValidationError: ('Validation Error', 'VALIDATION_ERROR', {'detail': [{'message': 'Invalid uuid', 'code': 'invalid_string', 'path': 'body.contractId'}]})` 

If you'd like to test for `ContractNotFoundError` you'd need to set up that up as a side effect in the httpx mock. 

Please let me know if you need an assist there and please make sure you pull in latest `main`.